args.py

```diff
@@ -65,20 +71,26 @@ def parse_args() -> Args:
         "--video-only", help="Only process videos", action="store_true"
     )
 
+    parser.add_argument(
+        "--video-async",
+        help="[EXPERIMENTAL] Convert several videos at once for a faster overall conversion.",
+        action="store_true",
+    )
```

Comment:
note: this conflicts with `--image-only`, since it has no effect on images. creating an exclusive group to map this seems annoying though, since image-only/video-only already conflict with each other